libs/designsystem/src/lib/components/tab-navigation/tab-navigation/tab-navigation.component.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  Component,
+  ContentChildren,
+  ElementRef,
+  EventEmitter,
+  Input,
+  OnDestroy,
+  Output,
+  QueryList,
+  ViewChild,
+} from '@angular/core';
+import { filter, fromEvent, map, Subject, takeUntil } from 'rxjs';
+import { WindowRef } from '@kirbydesign/designsystem/types';
+import { TabNavigationItemComponent } from '../tab-navigation-item/tab-navigation-item.component';
+
+const ARROW_LEFT = 'ArrowLeft';
+const ARROW_RIGHT = 'ArrowRight';
+
+@Component({
+  selector: 'kirby-tab-navigation',
+  templateUrl: './tab-navigation.component.html',
+  styleUrls: ['./tab-navigation.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TabNavigationComponent implements AfterViewInit, OnDestroy {
+  public readonly DEBOUNCE_TIME_MS = 250;
+  public readonly TAB_SELECTED_CLASS_NAME = 'selected';
+
+  @ViewChild('tabBar')
+  private tabBar: ElementRef<HTMLElement>;
+
+  @ContentChildren(TabNavigationItemComponent, { read: ElementRef })
+  private tabs: QueryList<ElementRef<HTMLElement>>;
+
+  private tabBarElement: HTMLElement;
+  private tabElements = new Array<HTMLElement>();
+  private destroyed$ = new Subject<void>();
+
+  @Input()
+  get selectedIndex(): number {
+    return this._selectedIndex;
+  }
+
+  set selectedIndex(index: number) {
+    if (index !== this._selectedIndex) {
+      this._selectedIndex = index;
+
+      setTimeout(() => {
+        this.selectTab(this.selectedIndex);
+        this.scrollToTab(this.selectedIndex);
+        this.selectedIndexChange.emit(this.selectedIndex);
+      });
+    }
+  }
+  private _selectedIndex = -1;
+
+  @Output()
+  selectedIndexChange = new EventEmitter<number>();
+
+  constructor(private window: WindowRef) {
+    /**/
+  }
+
+  ngAfterViewInit(): void {
+    this.tabBarElement = this.tabBar.nativeElement;
+    this.tabs.forEach((tab) => this.tabElements.push(tab.nativeElement));
+
+    this.initTabListeners();
+    setTimeout(() => {
+      this.scrollToTab(this.selectedIndex);
+    }, this.DEBOUNCE_TIME_MS);
+  }
+
+  initTabListeners(): void {
+    this.tabElements.forEach((tabElement, index) => {
+      fromEvent(tabElement, 'click')
```

Comment:
When I click anywhere outside of the tab-nav, the tab-button is no longer bold even though it is the selected tab.

Reply:
Postponed...